graphite/graphite.go

```diff
@@ -42,23 +47,55 @@ func (r *Request) Query(host string) (Response, error) {
 	if r.End != nil {
 		v.Add("until", fmt.Sprint(r.End.Unix()))
 	}
-	u := url.URL{
+	r.URL = url.URL{
 		Scheme:   "http",
 		Host:     host,
 		Path:     "/render/",
 		RawQuery: v.Encode(),
 	}
-	resp, err := DefaultClient.Get(u.String())
+	resp, err := DefaultClient.Get(r.URL.String())
 	if err != nil {
-		return Response{}, err
+		return nil, fmt.Errorf(requestErrFmt, r.URL.String(), "Get failed: "+err.Error())
```

Comment:
```
with String():
http://graphite/render/?format=json&from=1428778698&target=divideSeries%28stats.gauges.stats_buffer_space_mb.%2A._data_s3%2A%29&until=1429037898`

without:
{http  <nil> graphite /render/ format=json&from=1428778588&target=divideSeries%28stats.gauges.stats_buffer_space_mb.%2A._data_s3%2A%29&until=1429037788 }
```
I find the `String()` version clearer, better and also allows direct copy pasting.